mne/channels/layout.py

```diff
@@ -849,21 +852,26 @@ def _pair_grad_sensors_from_ch_names(ch_names):
     return grad_chs
 
 
-def _merge_grad_data(data):
-    """Merge data from channel pairs using the RMS.
+def _merge_grad_data(data, psd=False):
```

Comment:
i would prefer:

_merge_grad_data(data, method='rmse' | 'mean')

we may need to do a mean for something different than psd